### PR TITLE
Guide MagicMirror²

### PR DESCRIPTION
**Pull request by thomas-kammerer**
Here's a guide for MagicMirror² 
...for all people who need some craft projects while staying at home :)

### CODE REVIEW

**Comment by luto** [comment]
Thanks for your guide :)

We're a bit swamped with customer support currently, so it might take us a bit to review it. While we're busy, you can make sure that you're using [the snippets](https://github.com/Uberspace/lab/tree/master/source/includes) for domain setup, supervisord, mysql and other tasks. Sorry for the delay!

File: source/guide_magicmirror.rst [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+
+.. note:: Check the update feed_ regularly to stay informed about the newest version.
+
+Stop your service and repeat the installation step.
+
+.. _MagicMirror²: https://magicmirror.builders/
+.. _feed: https://github.com/MichMich/MagicMirror/releases
+
+----
+
```
Comment (nichtmax) [inline source/guide_magicmirror.rst:117]: I can't get it to work 😞

```
[moritz@stardust MagicMirror]$ npm start
npm WARN lifecycle The node binary used for scripts is /bin/node but npm is using /opt/nodejs12/bin/node itself. Use the `--scripts-prepend-node-path` option to include the path for the node binary npm was executed with.

> magicmirror@2.11.0 start /home/moritz/MagicMirror
> DISPLAY="${DISPLAY:=:0}" ./node_modules/.bin/electron js/electron.js

/home/moritz/MagicMirror/node_modules/electron/dist/electron: error while loading shared libraries: libgtk-3.so.0: cannot open shared object file: No such file or directory
npm ERR! code ELIFECYCLE
npm ERR! syscall spawn
npm ERR! file sh
npm ERR! errno ENOENT
npm ERR! magicmirror@2.11.0 start: `DISPLAY="${DISPLAY:=:0}" ./node_modules/.bin/electron js/electron.js`
npm ERR! spawn ENOENT
npm ERR! 
npm ERR! Failed at the magicmirror@2.11.0 start script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/moritz/.npm/_logs/2020-04-28T08_20_54_802Z-debug.log
```

Is it only me?

File: source/guide_openwebanalytics.rst [hunk 2]
```diff
@@ -0,0 +1,106 @@
+.. highlight:: console
```
Comment (nichtmax) [inline source/guide_openwebanalytics.rst:1]: That's the wrong pull request ☺️

File: source/guide_magicmirror.rst (hunk 1 above)
Comment (thomas-kammerer) [reply source/guide_magicmirror.rst:117]: Just fixed it, the command to start the server is `npm run server` not `npm start`.

**Comment by ebroda** [comment]
Hej,
is there are reason for removing the guide_openwebanalytics.rst?

**Comment by thomas-kammerer** [comment]
There's a sperate pull request for openwebanalytics:
https://github.com/Uberspace/lab/pull/619

**Pull request merged**